### PR TITLE
wikirunner/npmInstall

### PR DESCRIPTION
**Pull request by MarcelDiessner**
#178

### CODE REVIEW

File: runners/wikiConsole/templates/npmInstall.asciidoc [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Install npm
```
Comment (GuentherJulian) [inline runners/wikiConsole/templates/npmInstall.asciidoc:1]: You do not install npm in this step

File: runners/wikiConsole/templates/npmInstall.asciidoc [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+===========
+
+Prerequisites
+~~~~~~~~~~~~~~
+First make sure to have installed npm. We recommend to install the devonfwIDE which includes an npm installation. Alternatively, you can download npm https://www.npmjs.com/get-npm[here] and follow the installation instructions.
```
Comment (GuentherJulian) [inline runners/wikiConsole/templates/npmInstall.asciidoc:6]: npm and nodejs have to be installed

File: runners/wikiConsole/templates/npmInstall.asciidoc [hunk 3]
```diff
@@ -0,0 +1,19 @@
+Install npm
+===========
+
+Prerequisites
+~~~~~~~~~~~~~~
+First make sure to have installed npm. We recommend to install the devonfwIDE which includes an npm installation. Alternatively, you can download npm https://www.npmjs.com/get-npm[here] and follow the installation instructions. 
+Then, you need to be located in the project directory where the package.json file lies.
```
Comment (GuentherJulian) [inline runners/wikiConsole/templates/npmInstall.asciidoc:7]: You can use the npmInstall command without project directory, too. For example if you want to install a package globally

**Review by GuentherJulian — CHANGES_REQUESTED** [review]

File: runners/wikiConsole/templates/npmInstall.asciidoc [hunk 4]
```diff
@@ -0,0 +1,19 @@
+npmInstall
```
Comment (GuentherJulian) [inline runners/wikiConsole/templates/npmInstall.asciidoc:1]: The header should not only be the name of the command. It should describe shortly what is done

**Review by GuentherJulian — CHANGES_REQUESTED** [review]

**Review by denise-khuu — APPROVED** [review]

File: runners/wikiConsole/templates/npmInstall.asciidoc [hunk 5]
```diff
@@ -0,0 +1,19 @@
+Installation of packages with npm install
```
Comment (GuentherJulian) [inline runners/wikiConsole/templates/npmInstall.asciidoc:1]: I would find it better if you write the name of the package here if a package is given and write 'Install dependencies' if the user should install the packages from the package.json file

**Review by GuentherJulian — CHANGES_REQUESTED** [review]

**Review by GuentherJulian — APPROVED** [review]

**Pull request merged**